docs/faq.rst

```diff
@@ -37,6 +37,11 @@ which details your dependencies. See our guide on :ref:`guides/reproducible-buil
 You can also set your project documentation to install your project itself
 as a dependency.
 
+Additionally, your build may depend on extensions that require external,
+non-Python libraries to be installed. If you are using a :ref:`configuration
+file <config-file>` you can add libraries to the Ubuntu-based builder with
+``apt``.
```

Comment:
This would be great as a link to the section in config-file for apt install configuration option. It should be addressable using the auto section reference syntax